scripts/setup-annotations.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+
+# Determine PaSh top directory
+PASH_TOP=$(dirname "$(realpath "$0")")
```

Comment:
do we want to get the original PASH_TOP value if it is set already? Only if it is not set we want to set it based on this. Also, this setting here assumes we are calling the script from the root of the repo. I would suggest two modes:
1. if we have an extra argument with user provided path just execute there
2. if not, then check if PASH_TOP is set and if not print error message and exit. If it is set, we can install as a sibling